rocks.koreader.KOReader.yml

```diff
@@ -78,10 +77,17 @@ modules:
         commands:
           - |
             export KO_MULTIUSER=1
+            ARGS="$@"
 
             cd /app/koreader
+            RETURN_VALUE=85
+            while [ ${RETURN_VALUE} -eq 85 ]; do
+                ./reader.lua "${ARGS}"
+                RETURN_VALUE=$?
+                ARGS=""
+            done
+            exit ${RETURN_VALUE}
```

Comment:
```suggestion
            cd /app/koreader
            RETURN_VALUE=85
            while [ ${RETURN_VALUE} -eq 85 ]; do
                ./reader.lua "$@"
                RETURN_VALUE=$?
                set --
            done
            exit ${RETURN_VALUE}
```

Reply:
Otherwise you're concatenating multiple arguments into one the first time `./reader.lua` is called, and passing an empty arguments (instead of no arguments) the subsequent times.